pkgs/servers/monitoring/librenms/default.nix

```diff
@@ -0,0 +1,102 @@
+{ pkgs, stdenv, lib, fetchFromGitHub, unixtools, php82, python3, makeWrapper
+# run-time dependencies
+, graphviz
+, ipmitool
+, libvirt
+, monitoring-plugins
+, mtr
+, net-snmp
+, nfdump
+, nmap
+, rrdtool
+, system-sendmail
+, whois
+, dataDir ? "/var/lib/librenms", logDir ? "/var/log/librenms" }:
+
+
+let
+  package = (import ./composition.nix {
+    inherit pkgs;
+    inherit (stdenv.hostPlatform) system;
+    noDev = true; # Disable development dependencies
+    php = php82;
+  });
+
```

Comment:
Have you considered trying the new PHP builder which is FOD-style to reduce the footprint of that PR?

Reply:
Yes, but first I want to finish the distributed pollers feature.